spec/features/swagger_spec.rb

```diff
@@ -49,8 +49,8 @@
       it 'adds an Authorization header' do
```

Comment:
It seems from the OAuth2 RFC that only bearer (and MAC but MAC is not widely deployed) is supported. See https://tools.ietf.org/html/rfc6750. I'll return the tests to the original state and add an additional testing making the focus OAuth2.